src/hotspot/share/code/compressedStream.hpp

```diff
@@ -111,4 +112,122 @@ class CompressedWriteStream : public CompressedStream {
   }
 };
 
+class CompressedSparseData : public ResourceObj {
+protected:
+  int     _position; // current byte offset
+  size_t  _bit_position {0}; // current bit offset
+
+public:
+  CompressedSparseData(int position = 0) {
```

Comment:
As it is one-argument constructor, let it be `explicit CompressedSparseData`.

Reply:
ok